test/app/src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
So we now require React to be installed to run tests locally? This seems like a huge degrade in developer experience - as most devs do not add/edit e2e tests.

Can you explain the rational for using React versus our prior pure JS implementation?